inference-engine/ie_bridges/c/src/ie_c_api.cpp

```diff
@@ -335,11 +358,69 @@ IEStatusCode ie_core_read_network_from_memory(ie_core_t *core, const uint8_t *xm
     return status;
 }
 
+IEStatusCode ie_core_import_network(ie_core_t *core, const char *file_name, const char *device_name,
+        const ie_config_t *config, ie_executable_network_t **exe_network) {
+    IEStatusCode status = IEStatusCode::OK;
+
+    if (core == nullptr || file_name == nullptr || device_name == nullptr || exe_network == nullptr) {
+        status = IEStatusCode::GENERAL_ERROR;
+        return status;
+    }
+
+    try {
+        std::map<std::string, std::string> conf_map;
+        if (config != nullptr)
+            conf_map = config2Map(config);
```

Comment:
can the case `config == nullptr` be handled in `config2Map` itself? 
It would be nice to fix other places in this file too which check that config is mandatory.

Reply:
fixed